WireUI/Sources/WireIndividualToTeamMigrationUI/Views/ConfirmationView.swift

```diff
@@ -26,22 +26,42 @@ struct ConfirmationView: View {
         case `continue`
     }
 
+    let termsOfUseURL: String
+    let privacyPolicyURL: String
     let actionCallback: (Action) -> Void
     @State private var migrationConfirmed: Bool = false
     @State private var termsAccepted: Bool = false
 
     var body: some View {
         VStack(alignment: .leading, spacing: 56) {
-            Text(String.localized(key: "individualToTeam.confirmation.body", bundle: .module))
-                .wireTextStyle(.body1)
+            VStack(alignment: .leading) {
+                HStack(alignment: .top, spacing: 8) {
+                    Text("•")
```

Comment:
Please use the `verbatim` argument and remove `" *"` and `"•"` from the string catalog.
```suggestion
                    Text(verbatim: "•")
```